graphql_api/types/commit/commit.py

```diff
@@ -354,3 +362,120 @@ def resolve_bundle_status(commit: Commit, info) -> Optional[CommitStatus]:
 @sync_to_async
 def resolve_coverage_status(commit: Commit, info) -> Optional[CommitStatus]:
     return commit_status(commit, CommitReport.ReportType.COVERAGE)
+
+
+@commit_bindable.field("coverage")
+def resolve_commit_coverage(commit, info):
+    return commit
+
+
+@commit_bindable.field("bundleAnalysis")
+def resolve_commit_bundle_analysis(commit, info):
+    return commit
+
+
+### Commit Coverage Bindable ###
+
+
+@commit_coverage_bindable.field("totals")
+def resolve_coverage_totals(commit, info):
```

Comment:
If it's not a big deal, it'd be nice to throw some types on these (e.g., `def resolve_coverage_totals(commit: Commit, info: GraphQLResolveInfo) -> Dict[str, Any]:
` (not sure if that's 100% right))

Side note - I was exploring a little what it'd look like to define a type for the `Any` info.context using Python generics. I can use this as a testcase for that effort! (tracked [here](https://github.com/codecov/engineering-team/issues/2598))